### ISSUE TITLE
AVT 1 - Carbon X Tag Component DAP Violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1834]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 1 Run DAP on the example and there is 1 violation. See screenshot.
<img width="1153" alt="screen shot 2019-02-15 at 8 25 55 am" src="https://user-images.githubusercontent.com/21676914/52862695-763acb80-30fb-11e9-8fae-6c175501e13d.png">
Note: To fix violation add aria-disabled="true" to the last tag.

**Comment by snidersd** [comment]
Although the recommended fix will remove the violation it turns out that there is a DAP [GitHub issue 1184](https://github.ibm.com/IBMa/Tools-Rules-HTML/pull/1186) that is causing the violation in error and it should be fixed in the next DAP release expected later this month.  This should be re-tested with the new ruleset.

**Comment by dakahn** [comment]
Retested with the latest build and the error is resolved 👍
